sim/hunter/carve.go

```diff
@@ -60,7 +60,11 @@ func (hunter *Hunter) registerCarveSpell() {
 			curTarget := target
 			for idx := range results {
 				baseDamage := spell.Unit.MHNormalizedWeaponDamage(sim, spell.MeleeAttackPower())
-				results[idx] = spell.CalcDamage(sim, curTarget, baseDamage, spell.OutcomeMeleeWeaponSpecialHitAndCrit)
+				if curTarget == target {
+					results[idx] = spell.CalcDamage(sim, curTarget, baseDamage * 1.5, spell.OutcomeMeleeWeaponSpecialHitAndCrit)
+				} else {
+					results[idx] = spell.CalcDamage(sim, curTarget, baseDamage, spell.OutcomeMeleeWeaponSpecialHitAndCrit)
+				}
```

Comment:
We could just mod the base damage to save a bit of extra code
```suggestion
				if curTarget == target {
					baseDamage *= 1.5
				}
				results[idx] = spell.CalcDamage(sim, curTarget, baseDamage, spell.OutcomeMeleeWeaponSpecialHitAndCrit)
				}
```

Reply:
Looks good to me, I'll make that change in both locations